src/domains/profile/pages/ImportProfile/ProfileFormStep.tsx

```diff
@@ -84,6 +85,10 @@ const CreateProfileForm = ({
 		}
 	}, []); // eslint-disable-line react-hooks/exhaustive-deps
 
+	useEffect(() => {
+		trigger("confirmPassword");
+	}, [watchedPassword, trigger]);
```

Comment:
```suggestion
	useEffect(() => {
		if (confirmPassword) {
			trigger("confirmPassword");
		}
	}, [watchedPassword, trigger]); // eslint-disable-line react-hooks/exhaustive-deps
```
still needs this, otherwise

https://user-images.githubusercontent.com/24996341/127123048-e6fa3ad3-970e-4a9e-9415-5e04fcd3cd4e.mp4

Reply:
I'm actually thinking that having `getValues` back could be cleaner:
1. It is not reactive, and allows us to not include `confirmPassword` in the dependencies list
2. I don't think it's good to igore warnings (react-hooks/exhaustive-deps) unless it's really necessary

Does it make sense?

Reply:
1- yeah `getValues` is fine
2- In some cases, to avoid calling an infinite loop (or endless loop), it is ok to ignore this error and using the dependencies that we want.
didn't find the original article that I'd read about this, but [this article is good](https://blog.logrocket.com/guide-to-react-useeffect-hook/#:~:text=Finally%2C%20be%20aware%20that%20the%20plugin%20is%20not%20omniscient) too